app/routes/auth.callback.tsx

```diff
@@ -0,0 +1,73 @@
+import { redirect, type LoaderFunctionArgs } from "@remix-run/node";
+import {
+  createServerClient,
+  parse,
+  parseCookieHeader,
+  serializeCookieHeader,
+} from "@supabase/ssr/src";
+
+
+export async function loader({ request }: LoaderFunctionArgs) {
+  const requestUrl = new URL(request.url);
+  const code = requestUrl.searchParams.get("code");
+  const next = requestUrl.searchParams.get("next") || "/";
+  const headers = new Headers();
```

Comment:
**issue**: Remove console logs like `console.log("kakasraka: ", requestUrl);` to clean up the code for production.